src/deepsparse/transformers/pipelines.py

```diff
@@ -1404,7 +1405,7 @@ def process_dataset(
         batch_size=batch_size,
         task=task,
     )
-
+    pipeline_object = fix_numpy_types(pipeline_object)
```

Comment:
let's add a comment here to specify that `fix_numpy_types` is just wrapping the object

Reply:
Done!